format_and_overlap_determination.md

```diff
@@ -0,0 +1,118 @@
+# Description of Format and Overlap Determinations #
+This attempts to describe what this system actually does. This does not attempt to exhaustively describe all corner cases or emergent properties.
+
+## Definitions ##
+**Format**
+: mono (spm), multi (mpm), serial (ser)
+
+**Cluster**
+: Collection of HT Items, Holdings, and Concordance pairs united by a list of OCNs determined to be equivalent.
+
+**Bib Record**
+: A preferred record from the Zephir clustering system containing one or more HT Items. Has a single Hathitrust record ID.
+
+**HT Item**
+: Volume ID and associated metadata taken from the Hathifiles. 
+
+**Print Serials List**
+: List of Record Ids determined to be serials by some unknown UMich process. List is limited to those held by UMich.
```

Comment:
How do we get this list?  How often?

Reply:
Margaret Kelly (megrust@umich.edu) emails this as a .txt file to Martin & Josh at the end of each month. We could request instead that it be uploaded to a certain dropbox location.

Reply:
I think we should say:

- we get the file from AIM (do not specify individuals)
- how the file is generated is not in scope for this document

Reply:
We were told that the latest file (end of May 2021) would be the last file Margaret would be able to produce, as Umich is switching from Aleph to Alma (and presumably this report was based on Aleph).